src/Grpc.AspNetCore.Server/Internal/HttpContextServerCallContext.cs

```diff
@@ -130,5 +133,51 @@ protected override Task WriteResponseHeadersAsyncCore(Metadata responseHeaders)
 
             return HttpContext.Response.Body.FlushAsync();
         }
+
+        private DateTime GetDeadline()
```

Comment:
Curious where the logic to parse the deadline was specified? Was there a document on this I missed?

Reply:
https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md

Timeout → "grpc-timeout" TimeoutValue TimeoutUnit
TimeoutValue → {positive integer as ASCII string of at most 8 digits}
TimeoutUnit → Hour / Minute / Second / Millisecond / Microsecond / Nanosecond
Hour → "H"
Minute → "M"
Second → "S"
Millisecond → "m"
Microsecond → "u"
Nanosecond → "n"